ouroboros-consensus/src/Ouroboros/Consensus/Crypto/DSIGN/Cardano.hs

```diff
@@ -66,7 +66,7 @@ instance DSIGNAlgorithm CardanoDSIGN where
     type ContextDSIGN CardanoDSIGN = (ByronConfig, VerKeyDSIGN CardanoDSIGN)
 
     newtype VerKeyDSIGN CardanoDSIGN = VerKeyCardanoDSIGN VerificationKey
-        deriving (Show, Eq, Generic)
+        deriving (Eq, Generic, Ord, Show)
```

Comment:
I had a gut feeling about that. So should I just use an association list instead of `Map`, or is there a preferred alternate data structure already in the code base?

Reply:
I fixed this by using `KeyHash`. Thanks.